EKF/estimator_interface.cpp

```diff
@@ -576,6 +576,30 @@ bool EstimatorInterface::local_position_is_valid()
 	return !_deadreckon_time_exceeded;
 }
 
+bool EstimatorInterface::isOnlyActiveSourceOfHorizontalAiding(const bool aiding_flag) const
+{
+	return aiding_flag && !isOtherSourceOfHorizontalAiding(aiding_flag);
+}
+
+bool EstimatorInterface::isOtherSourceOfHorizontalAiding(const bool aiding_flag) const
+{
+	const int nb_sources = getNumberOfActiveHorizontalAidingSources();
+	return aiding_flag ? nb_sources > 1 : nb_sources > 0;
+}
+
+int EstimatorInterface::getNumberOfActiveHorizontalAidingSources() const
```

Comment:
Auxiliary Velocity would also be an horizontal aiding source, tough it is not actively used at the moment.

Reply:
Yes, the good thing is that we'll just have to add it here instead of in 20 places